specification/resourcegraph/resource-manager/readme.md

```diff
@@ -46,7 +46,9 @@ These settings apply only when `--tag=package-2021-03` is specified on the comma
 
 ``` yaml $(tag) == 'package-2021-03'
 input-file:
+  - Microsoft.ResourceGraph/preview/2020-09-01-preview/resourcechanges.json
   - Microsoft.ResourceGraph/stable/2021-03-01/resourcegraph.json
+  - Microsoft.ResourceGraph/preview/2020-04-01-preview/resourceshistory.json
 ```
```

Comment:
Instead of adding these two preview swaggers into this stable tag (which makes a lot differences in go SDK generation by making this from stable to preview), could we leave this tag unchanged but introduce a new tag `package-2021-03-preview` to have these three swaggers in it?

Reply:
Agree with @ArcturusZhang

Reply:
Thank you addressed the feedback.